xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxBXA.h

```diff
@@ -56,17 +56,19 @@ class CDVDDemuxBXA : public CDVDDemux
 
   bool Open(CDVDInputStream* pInput);
   void Dispose();
-  void Reset();
-  void Abort();
-  void Flush();
-  DemuxPacket* Read();
+
+  // implementation of CDVDDemux
+  virtual void Reset() override;
+  virtual void Abort() override;
+  virtual void Flush() override;
+  virtual DemuxPacket* Read() override;
   bool SeekTime(double time, bool backwards = false, double* startpts = NULL) override { return false; }
-  void SetSpeed(int iSpeed) {};
-  int GetStreamLength() { return (int)m_header.durationMs; }
+  virtual void SetSpeed(int iSpeed) override {}
```

Comment:
I don't kill warnings for the sake of killing warnings. We can take the opportunity to improve things that pop up here or just do nothing.
SetSpeed obvously was hacked into the interface at some time for a special case. Now we can take corrective action.